packages/esm-patient-registration-app/src/patient-registration/patient-registration-utils.ts

```diff
@@ -124,7 +124,7 @@ export function getFormValuesFromFhirPatient(patient: fhir.Patient) {
   result.additionalFamilyName = additionalPatientName?.family;
 
   result.gender = capitalize(patient.gender);
-  result.birthdate = patient.birthDate ? (new Date(patient.birthDate) as any) : undefined;
+  result.birthdate = patient.birthDate ? (parseDate(patient.birthDate) as any) : undefined;
```

Comment:
It’d be nice if we didn’t have to cast to `any`

```suggestion
  result.birthdate = patient.birthDate ? parseDate(patient.birthDate) : undefined;
```

Although I suspect this won’t work but because of the `undefined`…